test/testdata/circular-is.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
The word I've heard from on high is to just use YAML everywhere. If we want examples of JSON, we can have them in actual examples (under `examples/...`) but for stuff devs are working with the decision was YAML.